migrations/20231007120000_create_tiny_url.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+
+CREATE TABLE tiny_url (
+  id bigserial PRIMARY KEY,
+  org_url text NOT NULL,
+  short_url text UNIQUE NOT NULL,
+  comment text,
+  user_id int NOT NULL,
```

Comment:
- should we add a foreign key constraint to the users table ?

Reply:
added